h/static/scripts/directive/test/annotation-test.coffee

```diff
@@ -691,6 +751,44 @@ describe("AnnotationController", ->
     it("creates the directive without crashing", ->
       createAnnotationDirective({})
     )
+
+    it("sets the permissions of new annotations from local storage", ->
```

Comment:
here are only two cases here so it doesn't make much difference, but in general I prefer that when you have several tests with very similar logic, they be expressed as:

```
var cases = [...]
cases.forEach(function (case) {
it('does something for case ' + case.name, function () {
  // create variant using the case
});
}
```